src/main/java/planes/ExperimentalPlane.java

```diff
@@ -0,0 +1,48 @@
+package planes;
+
+import models.ClassificationLevel;
+import models.ExperimentalType;
+
+import java.util.Objects;
+
+public class ExperimentalPlane extends Plane {
+
+    private final ExperimentalType type;
+    private ClassificationLevel classificationLevel;
+
+    public ExperimentalPlane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity, ExperimentalType type, ClassificationLevel classificationLevel) {
+        super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
+        this.type = type;
+        this.classificationLevel = classificationLevel;
+    }
+
+    public ClassificationLevel getClassificationLevel() {
+        return classificationLevel;
+    }
+
+    public void setClassificationLevel(ClassificationLevel classificationLevel) {
+        this.classificationLevel = classificationLevel;
+    }
+
+    public ExperimentalType getType() {
+        return type;
+    }
+
+    @Override
+    public String toString() {
+        return super.toString().replace("}",
```

Comment:
This can be on the same line. Also, you can use String.format() method for better readability - "String.format(", type=%s, classification level=%s}", type, classificationLevel)"
(please apply the same change for other toString() methods in Plane classes)

Reply:
done